streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -36,6 +36,7 @@
     private final Serde<K> keySerde;
     private final Serde<V> valueSerde;
     private final boolean repartitionRequired;
+    private final String repartitionTopicName;
```

Comment:
nit: the use specified name is part of the repartition topic only -- we should rename this to `name` or `userName`?

Reply:
ack